eth/tracers/firehose.go

```diff
@@ -601,6 +629,14 @@ func (f *Firehose) reorderCallOrdinals(call *pbeth.Call, ordinalBase uint64) (or
 	return call.EndOrdinal
 }
 
+func (f *Firehose) OnClose() {
+	log.Info("Firehose closing...")
+	if f.concurrentBlockFlushing {
+		log.Info("Closing channel: flushing the remaining blocks to firehose")
```

Comment:
Since it's going to be "public", they should all start with `Firehose` the `log.Info` 
```suggestion
		log.Info("Firehose closing, flushing queued blocks to standard output")
```